netlogo-gui/src/main/lab/gui/RunOptionsDialog.scala

```diff
@@ -16,9 +16,14 @@ class RunOptionsDialog(parent: java.awt.Dialog,
     private val prefs = Preferences.userNodeForPackage(RunOptionsDialog.this.getClass)
     def spreadsheet = prefs.getBoolean("spreadsheet", true)
     def table = prefs.getBoolean("table", false)
+    /* add two more boolean property for disabling view,plots and monitor update */
+    def update_view = prefs.getBoolean("update_view",false)
+    def update_plots_monitor = prefs.getBoolean("update_plots_monitor",false)
```

Comment:
In Java/Scala, variable names (and symbolic names like the string names here) are `camelCased` and not `snake_cased`, as here.